x-pack/test/functional/apps/infra/constants.ts

```diff
@@ -40,3 +40,5 @@ export const ML_JOB_IDS = [
   'kibana-metrics-ui-default-default-k8s_network_in',
   'kibana-metrics-ui-default-default-k8s_memory_usage',
 ];
+
+export const KUBERNETES_TOUR_STORAGE_KEY = 'isKubernetesTourSeen';
```

Comment:
To avoid duplicating this constant, I would export it in the Kubernetes Tour component and use it in the functional tests. Something like this, *I think*, would work
```ts
import { KUBERNETES_TOUR_STORAGE_KEY } from '@kbn/infra-plugin/public/pages/metrics/inventory_view/components/show_kubernetes_tour';
```

Reply:
That worked, thanks!